pynumaflow/reducestreamer/servicer/task_manager.py

```diff
@@ -261,7 +261,7 @@ async def process_input_stream(self, request_iterator: AsyncIterable[reduce_pb2.
             # Once all tasks are completed, senf EOF the global result queue
             await self.global_result_queue.put(STREAM_EOF)
         except Exception as e:
-            err_msg = "Reduce Streaming Error: %r" % e.__str__()
+            err_msg = f"Reduce Streaming Error: {str(e)}"
```

Comment:
Can you do repr here? Gives better information.

Reply:
Done